packages/cubejs-schema-compiler/src/scaffolding/formatters/BaseSchemaFormatter.ts

```diff
@@ -146,36 +139,45 @@ export abstract class BaseSchemaFormatter {
           },
         }))
         .reduce((a, b) => ({ ...a, ...b }), {}),
-      measures: tableSchema.measures
+      dimensions: tableSchema.dimensions
         .map((m) => ({
           [this.memberName(m)]: {
             sql: this.sqlForMember(m),
             type: m.type ?? m.types[0],
             title: this.memberTitle(m),
+            [this.options.snakeCase ? 'primary_key' : 'primaryKey']: m.isPrimaryKey
+              ? true
+              : undefined,
           },
         }))
-        .reduce((a, b) => ({ ...a, ...b }), {
-          count: {
-            type: 'count',
-          },
-        }),
-      dimensions: tableSchema.dimensions
+        .reduce((a, b) => ({ ...a, ...b }), {}),
+      measures: tableSchema.measures
         .map((m) => ({
           [this.memberName(m)]: {
             sql: this.sqlForMember(m),
             type: m.type ?? m.types[0],
             title: this.memberTitle(m),
-            [this.options.snakeCase ? 'primary_key' : 'primaryKey']: m.isPrimaryKey
-              ? true
-              : undefined,
           },
         }))
-        .reduce((a, b) => ({ ...a, ...b }), {}),
+        .reduce((a, b) => ({ ...a, ...b }), {
+          count: {
+            type: 'count',
+          },
+        }),
+        
       ...(this.options.snakeCase
         ? Object.fromEntries(
           Object.entries(contextProps).map(([key, value]) => [toSnakeCase(key), value])
         )
         : contextProps),
+
+      [this.options.snakeCase ? 'pre_aggregations' : 'preAggregations']: new ValueWithComments(
```

Comment:
@vasilev-alex Do we need `this.options.snakeCase` anymore? Can't we go with snake case at all times?

Reply:
have to keep it for backwards compatibility